modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Integrating software RAID arrays with {lvms}
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID arrays using the `mdadm` utility, and integrate the RAID arrays with {lvms}.
+You can integrate the RAID arrays with {lvms} while creating the LVM cluster. 
+
+.Prerequisites
+
+* You have created a software RAID during the {product-title} installation.
++
+[NOTE]
+====
+You can create the RAID arrays only during the installation of {product-title}. For information about creating RAID arrays in {product-title}, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/installing/installation-configuration#installation-special-config-raid_installing-customizing[Configuring a RAID-enabled data volume]
+====
+
+* You have installed {lvms}.
+
+.Procedure
+
+. Open the `LVMCluster` custom resource (CR) YAML file.
++
+[NOTE]
+====
+If you have created the LVM cluster, then you can edit the existing `LVMCluster` CR. However, it is recommended to integrate the RAID arrays with the {lvms} while creating the LVM cluster.
+====
+
+. Add the path to the RAID array in the `deviceSelector` section in `LVMCluster` CR.
++
+.Example of RAID array paths in deviceSelector
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: my-lvmcluster
+spec:
+  storage:
+    deviceClasses:
+    - name: vg1
+      default: true
+      deviceSelector: <1>
+        paths:
+        - /dev/md/md-var <2>
+        - /dev/md0 <3>
+        optionalPaths:
+        forceWipeDevicesAndDestroyAllData: false
+      thinPoolConfig:
+        name: thin-pool-1
+        sizePercent: 90
+        overprovisionRatio: 10
+----
+<1> Contains the paths to the devices that are used to create the LVM volume group. You can specify the device paths in `paths`, `optionalPaths`, or both. You cannot remove or replace a device after adding it to the LVM cluster.
+<2> Example of a path to the RAID array created using `mdadm` utility. In this example, `md-var` is the RAID array.
```

Comment:
If these values would change the example might be better to change the values to <user_input_values>?

Reply:
The LVMCluster CR sample contains specific examples of other fields. Hence, added specific device path examples to maintain consistency.